spa/src/components/pageEditor/editInterface/EditTabHeader.styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const Prompt = styled('span')`
```

Comment:
When we style a html base component, we usually use the syntax 
```suggestion
export const Prompt = styled.span`
```

Reply:
👍 updated in 88845bde99e18e6f4d35f74d2c059d51d08e6612 and I also added an item to our coding style guide about this.